CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ PrestaShop 1.6.x and 1.7.x
 
 # Changelog
 
+## [3.3.5] - 2020-12-03
+### Added
+- Handled fail statuses properly
```

Comment:
Please be precise when defining a change log:
- Handle failed status as a failed order

I am guessing it is not all failed status that were not handled, right? Is there any particular scenario that has been fixed?

What is the new representation of the order in the CRM? Failed? Declined? Errored?

Reply:
So there are two statuses that were not handled from the plugin side "failed" and "error". which are now updated. I have updated the changelogs as well.